contact.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="ja">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Cresta Design</title>
+  <link rel="stylesheet" href="css/reset.css" />
+  <link rel="stylesheet" href="css/style.css" />
+  <link
+    href="https://fonts.googleapis.com/css?family=Noto+Serif+JP:400,700|Noto+Serif:400,700,700i&display=swap&subset=japanese"
+    rel="stylesheet">
+</head>
+
+
+<body id="body">
+
+  <div class="header_ham" id="jsham">
+    <span></span>
+    <span></span>
+    <span></span>
+  </div>
+
+  <header class="header">
+    <div class="fv-contact">
+
+
+
+      <div class="header-inner-contact hidden" id="header">
+        <div class="inner header-list">
+          <div class="header-title">
+            <h1>Cresta Design</h1>
+          </div>
+
+          <nav class="header-nav" id="jsnav">
+            <ul class="nav-list" id="jsnavul">
+              <li class="nav-list-item"><a href="index.html#Concept">Concept</a></li>
+              <li class="nav-list-item"><a href="index.html#Service">Service</a></li>
+              <li class="nav-list-item"><a href="index.html#Works">Works</a></li>
+              <li class="nav-list-item"><a href="contact.html">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div class="contact-title">
+        <h2>Contact</h2>
+      </div>
+
+      <div class="contact-phrase-white inner">
+        <p>Contact us</p>
+      </div>
+
+    </div>
+
+  </header>
+
+  <form action="#" class="form">
+
+    <div class="checkbox">
+      <p>お問い合わせ内容</p>
+      <label><input type="checkbox" name="contact" value="document"> 資料請求</label>
+
+      <label><input type="checkbox" name="contact" value="consultation"> お電話でのご相談希望</label>
+
+      <label><input type="checkbox" name="contact" value="application"> 申し込み</label>
+
+    </div>
+
+
+
+    <div class="form-part">
+      <label for="name">担当者名</label>
+      <input type="text" id="name" name="name">
+    </div>
+
+    <div class="form-part">
+      <label for="tell">電話番号</label>
+      <input type="tell" id="tell" name="your-tell">
+    </div>
+
+    <div class="form-part">
+      <label for="email">メールアドレス</label>
+      <input type="email" id="email" name="your-email">
+    </div>
+
+    <div class="form-part">
+      <label for="comment">その他</label>
+      <textarea name="comment" type="text" cols="30" rows="10"></textarea>
+    </div>
+
+    <div class="form-send">
+      <input type="submit" value="Submit" class="button button-yellow">
+    </div>
+  </form>
+
+
+  <footer>
+    <div class="footer">
+      <p>©︎cresta.design all rights reserved</p>

```

Comment:
[参考サイトです](https://creatorquest.jp/lessons/html/copyright/)
```suggestion
      <p lang="en"><small>&copy; cresta.design all rights reserved</small></p>

```

Reply:
ありがとうございます！

下記コミットで修正しました。

558083c